tests/test_transaction.py

```diff
@@ -215,6 +215,42 @@ def test_find_wheel_invalid_version():
     assert str(wheel.version) == "0.15.5"
 
 
+def test_yanked_version():
+    from micropip._vendored.packaging.src.packaging.requirements import Requirement
+    from micropip.transaction import find_wheel
+
+    versions = ["0.0.1", "0.15.5", "0.9.1"]
```

Comment:
I think we should add two more tests, one test for the case where one requests a yanked wheel explicitly (similar to the above point), and another case which checks if a warning was raised or not upon installing a yanked wheel (if you agree with my above suggestion, that is).

Reply:
Thanks. Updated test cases.